server/app.ts

```diff
@@ -5,10 +5,13 @@ import cors from 'cors';
 import morgan from 'morgan';
 import cookieParser from 'cookie-parser';
 import route from './routes/index';
+import { notFound, serverError } from './middlewares';
```

Comment:
I think if you use the same name convention for all functions or middlewares it will be better for understand the code
so there is middleware called `notFound` and another one called `serverError`, when anyone need to read this names he won't know that `notFound` used to handle 404 NOT FOUND
* **`serverError`**
* **`pageNotFoundError`**